src/Achievements/AchievementCategory.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import { Accordion, AccordionSummary, AccordionDetails, Typography } from "@mui/material";
+
+import { Achievement } from "./Achievements";
+
+interface IProps {
+  title: string;
+  achievements: { achievement: Achievement }[];
+  allAchievements?: { achievement: Achievement }[];
+  pad?: boolean;
+}
+
+function steamCount(achievements: { achievement: Achievement }[]): number {
+  return achievements.filter((entry) => !entry.achievement.NotInSteam).length;
+}
+
+export function AchievementCategory({
+  title,
+  achievements,
+  allAchievements,
+  pad,
+  children,
+}: React.PropsWithChildren<IProps>): JSX.Element {
+  // The 264px minWidth feels scuffed, but fixes an unknown edge case (Brought up in PR #1508).
+  return (
+    <Accordion defaultExpanded={!!allAchievements} disableGutters square sx={{ minWidth: "264px" }}>
```

Comment:
```suggestion
  /**
   * For each achievement, we need to display the icon and the detail on the same "row" (icon on the left and detail on
   * the right). When the viewport is too small, the detail part of some achievements is "moved" to a separate "row". It
   * looks like this:

<achievement 1>
  <icon><detail>
</achievement 1>
<achievement 2>
  <icon>
  <detail>
</achievement 2>
<achievement 3>
  <icon><detail>
</achievement 3>

   * Using "minWidth" fixes this issue by setting a min value for the width of each row.
   */
  return (
    <Accordion defaultExpanded={!!allAchievements} disableGutters square sx={{ minWidth: "645px" }}>
```
I guess that the edge case is what I wrote in the suggested change. Note that using 264px is not enough to fix this issue. Using 645px should be good enough.

Before:

![before](https://github.com/user-attachments/assets/b08a6e88-98bf-4dc1-b4c7-9b66c9d64e11)

After:

![after](https://github.com/user-attachments/assets/c2904ff5-edb9-4f63-bea8-a6bf475320f5)

There are other CSS tricks, but it's unnecessary for this PR. Maybe I'll use them in another PR. It's best to keep this one simple.